### ISSUE TITLE
Add loader

### ISSUE THREAD

**Issue by ashutoshkrris** [ashutoshkrris/Random-APIs#11]
### Is your feature request related to a problem? Please describe.
A clear and concise description of what the problem is. 
There is no way to understand if some data is loading

### Describe the solution you'd like
While the data is being fetched, a loader should be there

### Describe how this feature will be useful to our readers.
A clear and concise description of how this feature will be useful for our  other readers.

### Which Program are you part of?

- [x] Hacktoberfest21
- [ ] Any other

**Comment by everly-gif** [comment]
I would like to work on this, please assign me.

quick question:

 however , whenever I refresh the website , I don't see any loading issues it seems fine and the APIs are displaying the relevant output. Particularly  where is a loading delay?

**Comment by ashutoshkrris** [comment]
As the project continues to grow, a lot of APIs need to be fetched simultaneously.  In that case, we may require a loader.

**Comment by everly-gif** [comment]
Okay got it ! Thanks. I would like to work on this !

**Comment by ashutoshkrris** [comment]
Great @everly-gif 
Assigning it to you!

**Comment by everly-gif** [comment]
hey I have a question. So since you have used query selector and are appending the responses to the selected div. How can I go about this. I have created loading animation with CSS and before you call fetch I have set the response.innerHTML to the loader. when you append the response, ie: the line :  responsesBody.innerHTML += response; I need to remove the loader from it. In that case all the fetch calls will need an additional code of line. Am I going right about this?
